scripts/prove_rpc.sh

```diff
@@ -134,6 +135,7 @@ else
     echo "Proving blocks from ($START_BLOCK) to ($END_BLOCK)"
     command="cargo r --release --package zero --bin leader -- \
 --runtime in-memory \
+--use-test-config \
```

Comment:
not sure we want the script to always use the testing config, rather take it as optional argument perhaps?

Reply:
I assume we always use the testing config in CI tests, while our cron jobs use the standard config?

Reply:
I thought we had said in the dev sync we'd use a mix of both in CI? I'm fine either way though

Reply:
Done. Keep `Execute bash script to generate and verify a proof for a small block.` with default config.